### PR TITLE
[FrameworkBundle][HttpKernel] Document `collect_parameter`

### LINKED ISSUE #16047: [FrameworkBundle][HttpKernel] Add the ability to enable the profiler us…
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#43138
| PR author(s) | @dunglas
| Merged in    | 5.4

### PR DESCRIPTION
**Pull request by SzymonKaminski**
This PR aims to resolve #16047.

### CODE REVIEW

File: reference/configuration/framework.rst [hunk 1]
```diff
@@ -1298,6 +1298,19 @@ activate the data collectors manually::
...
+**type**: ``string`` **default**: ``null``
+
+This option allows to enable or disable the profiler on a per request basis
+using a query parameter, body parameter or a request attribute with the given name.
+If ``collect`` flag is set to ``true``, but the parameter exists in a request
+and has any value other than ``true``, ``yes``, ``on`` or ``1``,
+the request data will not be collected.
+If ``collect`` flag is set to ``false``, but the parameter exists in a request
+and has value of ``true``, ``yes``, ``on`` or ``1``, the request data will be collected.
+
```
Comment (OskarStark) [inline reference/configuration/framework.rst:1318]:
```suggestion

..versionadded:: 5.4

    The `collect_parameter` was introduced in Symfony 5.4.

```

File: reference/configuration/framework.rst [hunk 2]
```diff
@@ -1298,6 +1298,19 @@ activate the data collectors manually::
 
     $profiler->enable();
 
+collect_parameter
+.................
+
+**type**: ``string`` **default**: ``null``
+
+This option allows to enable or disable the profiler on a per request basis
```
Comment (OskarStark) [inline reference/configuration/framework.rst]: Are you sure this is the description for the collect**_parameter** option and not for the `collect` option above?

Comment (SzymonKaminski) [reply reference/configuration/framework.rst]: I've changed the first sentence, hopefully it's less confusing right now.

**Review by 94noni — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Szymon thanks for this nice addition ... and congrats on your first Symfony Docs contribution 🎉

**Pull request merged**